.changelog/122.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:improvement
+Update bootstrap configuration.
```

Comment:
Can this mention the rename, since there's a previous changelog entry about it? 
https://github.com/hashicorp/consul-dataplane/blob/main/.changelog/90.txt

Reply:
Sure will do!